README.md

```diff
@@ -1,9 +1,11 @@
 [![Build Status](https://travis-ci.org/ibm-watson-data-lab/shopping-list-vuejs-pouchdb.svg?branch=master)](https://travis-ci.org/ibm-watson-data-lab/shopping-list-vuejs-pouchdb)
 ![IBM Cloud Deployments](https://metrics-tracker.mybluemix.net/stats/4b4ca7dd406587d30b91b8ecff5eaf6b/badge.svg)
 
-# Create an Offline First Shopping List with Vue.js
+# Create an Offline First Shopping List with Vue.js and PouchDB
 
-In this code pattern, we will create an Offline First shopping list. Shopping List is an Offline First demo Progressive Web App built using [Vue.js](https://vuejs.org/). [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list).
+This code pattern is a reference implementation of an Offline First shopping list app, built as a Progressive Web App using [Vue.js](https://vuejs.org/) and [PouchDB](https://pouchdb.com/). [This app is part of a series of Offline First demo apps, each built using a different stack.](https://github.com/ibm-watson-data-lab/shopping-list).
```

Comment:
There are two periods at the end of the last sentence (one in the link and one outside of the link).